src/ProjectOrigin.Registry.IntegrationTests/Extensions/RandomExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace ProjectOrigin.Registry.IntegrationTests.Extensions
+{
+    public static class RandomExtensions
+    {
+        public static string GenerateString(this Random r, int length = 10)
+        {
+            var bytes = new byte[length];
+            r.NextBytes(bytes);
+            return Convert.ToBase64String(bytes);
+        }
+    }
+}
```

Comment:
Is this to simulate WalletDepositEndpoints?

Reply:
just a shorthand to easily generate base64 strings of certain byte lengths.